sympy/physics/control/lti.py

```diff
@@ -4263,3 +4263,134 @@ def is_controllable(self):
 
         """
         return self.controllability_matrix().rank() == self.num_states
+
+class PIDController(TransferFunction):
+    """
+    A PID Controller class that uses Parallel connection to combine the P, I, and D components.
+    Parameters
+    ==========
+    KP : Expr, Number
+        Proportional gain.
+    KI : Expr, Number
+        Integral gain.
+    KD : Expr, Number
+        Derivative gain.
+    TP : Expr, Number
```

Comment:
This is written as TF everywhere else so make it consistent. 
I was not aware of this being added to the implementation by the way. Why should it be considered ?

Reply:
I've gone through the documentation of MATLAB for it https://in.mathworks.com/help/control/ug/proportional-integral-derivative-pid-controllers.html, and It included Tf.

Reply:
Should I remove the Tf from here ?

Reply:
```suggestion
    TF : Expr, Number
```
No, I just wanted the argument name to be consistent with the code.

Reply:
That was my mistake, I've not noticed that thank for pointing out.